### PR TITLE
WIP: Release notes for 16.10

### PR DESCRIPTION
**Pull request by varshavaradarajan**
@ketan, is this okay?

### CODE REVIEW

File: source/partials/release_notes/_release-16.10.0.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<h4>Deprecations</h4>
+<p>Features that become superceeded or irrelevant become <em>deprecated</em>, and are scheduled to be removed in some timeframe.</p>
+<p>The following features have been deprecated, along with the timeframe when they will be removed</p>
+
+<ul>
+  <li><p>
+    As of release 16.10, version 2 of the <%= link_to_api 'agents', 'Agents' %> is deprecated.
+    This version of the API will be removed in 16.13 release (scheduled to be released in Dec 2016).
```
Comment (ketan) [inline source/partials/release_notes/_release-16.10.0.html.erb]: 16.12 you mean, or 17.1?

File: source/partials/release_notes/_release-16.10.0.html.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    Version 2 of this API is backward compatible, and users are encouraged to use it.
```
Comment (ketan) [inline source/partials/release_notes/_release-16.10.0.html.erb]: Version 3 and 4 are compatible. V2 is going to be deleted :)

File: source/partials/release_notes/_release-16.10.0.html.erb (hunk 1 above)
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-16.10.0.html.erb]: Won't we be making 4 more releases this year at least?

File: source/partials/release_notes/_release-16.10.0.html.erb (hunk 2 above)
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-16.10.0.html.erb]: Cool.  :)

File: source/partials/release_notes/_release-16.10.0.html.erb (hunk 1 above)
Comment (ketan) [inline source/partials/release_notes/_release-16.10.0.html.erb]: Lets make this 17.1, since there's not enough heads up for deprecation

**Pull request closed without merging**